modules/gdscript/gdscript_editor.cpp

```diff
@@ -2808,6 +2812,15 @@ static void _find_call_arguments(GDScriptParser::CompletionContext &p_context, c
 					r_arghint = _make_arguments_hint(info, p_argidx);
 				}
 
+				if (p_base.value.get_type() == Variant::NIL) {
+					if (!base.is_ref_counted()) {
+						Object *obj = base.get_validated_object();
+						if (obj != nullptr) {
+							memfree(obj);
```

Comment:
I'd say to use `memdelete_notnull` instead and not use the check against nullptr

Reply:
Is there maybe also some builtin, that handles refcounted correctly?